src/styles/components/Sponsor.module.css

```diff
@@ -1,35 +1,52 @@
 .container {
   display: flex;
-  padding: 0.5rem;
-  width: 9rem;
+  justify-content: center;
+  align-items: center;
+  width: 12rem;
   height: 8rem;
   border-radius: 1rem;
   background-color: #ffffff;
   position: relative;
   margin: 1rem;
+  z-index: 3;
   cursor: pointer;
-  display: flex;
-  border: none;
+  border: 4px solid #d9534f; /* Border color to match the screenshot */
```

Comment:
could you try to use the css color variables from global.css? this goes for all of the colors you have, try not to hard code (unless it doesn't exist in the globals.css file)